shortcuts

```diff
@@ -0,0 +1,4 @@
+sout - System.out.println()
```

Comment:
please check the format 
 https://github.com/ReDI-School/course-template/blob/master/glossary.md 

should be something similar, + in the readme there is an instruction to run it locally so you can see how your page looks like.